optuna/storages/_rdb/alembic/versions/v3.0.0.c.py

```diff
@@ -65,21 +85,28 @@ def upgrade():
         records = session.query(IntermediateValueModel).all()
         mapping = []
         for r in records:
-            float_type: IntermediateValueModel.FloatTypeEnum
+            value: float
             if np.isclose(r.intermediate_value, RDB_MAX_FLOAT) or np.isposinf(
                 r.intermediate_value
             ):
-                float_type = IntermediateValueModel.FloatTypeEnum.INF_POS
+                value = np.inf
             elif np.isclose(r.intermediate_value, RDB_MIN_FLOAT) or np.isneginf(
                 r.intermediate_value
             ):
-                float_type = IntermediateValueModel.FloatTypeEnum.INF_NEG
+                value = -np.inf
+            elif np.isnan(r.intermediate_value):
+                value = np.nan
             else:
-                continue
+                value = r.intermediate_value
+            (
+                sanitized_value,
+                float_type,
```

Comment:
"Internal" would be ambiguous because this representation is only for storage (i.e. this representation is not kept around in optuna). I think this is the main difference with categorical index (which is kept around in optuna), and is the reason I didn't want to use the term "internal".

Reply:
I don't have particular preference for "sanitized", but I could not come up with a better word. (Maybe `storable_value` would be better?)

Reply:
> "Internal" would be ambiguous because this representation is only for storage (i.e. this representation is not kept around in optuna). I think this is the main difference with categorical index (which is kept around in optuna), and is the reason I didn't want to use the term "internal".

Thank you for your explanation. In my understanding, `internal` just indicates that it requires the some sort of conversion, and I guess it matches this case. Since it converted by `_internal_value_to_stored_repr` method, `stored_value` may be more intuitive.

Reply:
I changed the name to `stored_value`.